packages/app/pages/index.tsx

```diff
@@ -2,939 +2,208 @@ import type { NextPage } from "next";
 import { useRouter } from "next/router";
 import Head from "next/head";
 import Link from "next/link";
-import React, { useCallback, useEffect, useMemo, useState } from "react";
-import { Contract, ContractSendMethod } from "web3-eth-contract";
-import { TransactionReceipt } from "web3-core";
-import {
-  Modal,
-  Input,
-  Button,
-  Label,
-  Box,
-  Toast,
-  Checkbox,
-} from "@cabindao/topo";
+import React, { useCallback, useMemo } from "react";
+import { Button } from "@cabindao/topo";
 import { styled } from "../stitches.config";
-import passportFactoryJson from "@cabindao/nft-passport-contracts/artifacts/contracts/PassportFactory.sol/PassportFactory.json";
-import passportJson from "@cabindao/nft-passport-contracts/artifacts/contracts/Passport.sol/Passport.json";
-import {
-  contractAddressesByNetworkId,
-  getAbiFromJson,
-  networkNameById,
-  shimmer,
-  toBase64,
-} from "../components/constants";
-import {
-  Link1Icon,
-  Pencil2Icon,
-  Share1Icon,
-  TrashIcon,
-  ArrowUpIcon,
-  OpacityIcon
-} from "@radix-ui/react-icons";
 import {
   useAddress,
   useChainId,
   useConnect,
-  useWeb3,
+  useDisconnect,
+  useDisplayAddress,
   Web3Provider,
 } from "../components/Web3Context";
-import axios from "axios";
+import MembershipTabContent from "../components/MembershipTabContent";
 import UsersTabContent from "../components/UsersTabContent";
-import { ipfsAdd, resolveAddress } from "../components/utils";
 import SettingsTabContent from "../components/SettingsTabContent";
-import IpfsImage from "../components/IpfsImage";
 import ManageTabContent from "../components/ManageTabContent";
-import Papa from "papaparse";
-import BN from "bn.js";
+import { networkNameById } from "../components/constants";
 
-const DRAWER_WIDTH = 255;
+const DRAWER_WIDTH = 200;
 const HEADER_HEIGHT = 64;
 
 const TabContainer = styled("div", {
-  minHeight: 64,
-  padding: 20,
+  margin: "12px 0",
   cursor: "pointer",
+  textTransform: "uppercase",
+  fontWeight: 600,
+  fontSize: "16px",
+  lineHeight: "20.8px",
+  variants: {
+    active: {
+      true: {
+        color: "$wheat",
+      },
+      false: {
+        color: "$sand",
+      },
+    },
+  },
```

Comment:
there are both css changes and the MembershipTabContent move change here. It's probably easier to review by looking at the deployed branch instead of line by line here